2018-westphal-antpc.tex

```diff
@@ -73,159 +77,175 @@ \section{Background: Pyroprocessing}
+significantly hindered. Cathode arrangement and anode rotation speed also affect the collection of uranium 
+dendrites \cite{lee_advanced_2008}. The addition of a central stirrer allows mixing of uranium dendrites stuck on 
+the bottom of the vessel, improving separation efficiency and increasing throughput. 
+
+\begin{figure}[ht]
+	\centering
+	\includegraphics[width=0.9\linewidth]{refining}
+	\caption{A material balance over the electrorefining sub-process, including observables.}
+	\label{fig:refining}
+\end{figure}
+
+The electrorefining process also produces the fission product waste stream which requires monitoring. The following products are produced 
+and tracked in this step: Tc, Ag, Pd, Rh, Ru, Mo and Zr \cite{flowsheet_1998}.
+
 \subsection{Electrowinning}
 
-The molten salt contains TRUs from electrorefining and are separated through electrowinning with trace uranium quantities. 
+The molten salt contains TRUs from electrorefining that are separated through electrowinning with trace uranium quantities. 
 With a temperature of 500$^{\circ}$C there is approximately 99 wt\% reduction in actinides and lanthanides \cite{flowsheet_1998}. 
+Throughput is also dependent on material choice for the inert electrodes whose operating voltages vary, impacting separation 
+efficiency \cite{koyama_development_2012}. A shroud surrounds the anode to provide a path for O$^{2-}$ ions to the anode and 
+prevent Cl$_2$ from corroding the anode \cite{kim_development_2013,choi_electrochemical_2015}. Optimum operating current 
+depends on the material choice for the anode shroud since a nonporous shroud limits ion pathways to the points of contact 
+with the anode. Higher porosity corresponds to free ion paths and a higher current. With a higher current the separation 
+time for reduction and winning are reduced \cite{choi_electrochemical_2015}.
+
+\begin{figure}[ht]
+	\centering
+	\includegraphics[width=1\linewidth]{winning}
+	\caption{A material balance over the electrowinning sub-process, including observables.}
+	\label{fig:winning}
+\end{figure}
+
+Figure \ref{fig:winning} shows that electrowinning produces Cs waste similar to electroreduction except in reduced quantities. Any fission products remaining in the salt from electrorefining are also removed here. Outside of physically tracked quantities, facility power can also be monitored to observe the use of current to increase throughput. 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Method: \Cyclus Simulation}
-The separations facility provided by the \Cycamore library expansion is used as an initial model of a simple PRIDE facility. 
-The separations archetype allows for the declaration of a feed stream and requires the user-definition of facility efficiencies. 
+The separations facility provided by the \Cycamore library is used as an initial model of a simple PRIDE facility. 
```

Comment:
The pride facility has not been explained and KAERI is no longer mentioned or cited (those parts were deleted?). This is not good.